tests/integration/integration.spec.js

```diff
@@ -83,6 +83,44 @@ describe('FVL integration', () => {
     expect(wrapper.find('span').text()).toBe('')
   })
 
+  it('renders label in error messages', async () => {
+    const schema = {
+      firstName: {
+        label: 'First Name',
+        component: FormText,
+        validations: yup.string().required().label('First Name')
+      }
+    }
+
+    const SchemaWithValidation = SchemaFormFactory([veeValidatePlugin()])
+
+    const wrapper = mount({
+      template: `
+        <SchemaWithValidation :schema="schema" />
+      `,
+      components: {
+        SchemaWithValidation
+      },
+      setup () {
+        const formData = ref({})
+        useSchemaForm(formData)
+
+        return {
+          schema
+        }
+      }
+    })
+
+    const input = wrapper.findComponent(FormText)
+    input.setValue('')
+    await flushPromises()
+    console.log(wrapper.find('span').text())
+    expect(wrapper.find('span').text()).toMatch(/First Name/)
```

Comment:
nitpick: Searching for a `span` on the test is very deeply tied to the default implementation of vee-validate which would break if it changes. Lets change line `91` to say `label('First name is required')` and simply `wrapper.text()` `.toContain('First name is required')` to add some more clarity.

This would also apply to the second test

Reply:
@marina-mosti wrapper.text() works just fine, without selecting the `span` first, so I'm happy to remove it.  I was just going with the existing pattern (there are 16 instances of it in the file).

As for changing line 91... The `label()` method is used to pass the value of the label to be interpolated into the `errorMessage` string. If I do as you suggest, the resulting `errorMessage` is "First Name is a required field is a required field" which I don't think is what is needed.

Let me know if this makes sense!

Reply:
Right, derp. Thanks for pointing out the existing patter, I'll take a look at it later but I guess no big deal :)